metadata_mapper/utilities.py

```diff
@@ -0,0 +1,138 @@
+import boto3
+import importlib
+import json
+import os
+
+from typing import Union
+
+import settings
+
+from mappers.mapper import Vernacular
+
+
+def import_vernacular_reader(mapper_type):
+    """
+    accept underscored_module_name_prefixes
+    accept CamelCase class name prefixes split on underscores
+    for example:
+    mapper_type | mapper module       | class name
+    ------------|---------------------|------------------
+    nuxeo       | nuxeo_mapper        | NuxeoVernacular
+    content_dm  | content_dm_mapper   | ContentDmVernacular
+    """
+    mapper_parent_modules, snake_cased_mapper_name = mapper_type.split(".")
+
+    mapper_module = importlib.import_module(
+        f"mappers.{mapper_parent_modules}.{snake_cased_mapper_name}_mapper",
```

Comment:
Should read: `f"mappers.{'.'.join(mapper_parent_modules)}.{snake_cased_mapper_name}_mapper",`